content/vue/step02.md

```diff
@@ -0,0 +1,124 @@
+{{#template name="vue-step02"}}
+
+# In a hurry?
+
+If you don't have time to follow this detailed tutorial right now, you have two options.
+
+To skip to the finished application explained by this tutorial, run the following commands:
+
+```sh
+git clone git@github.com:meteor/simple-todos-vue.git
+cd simple-todos-vue
+meteor npm install
+meteor
+```
+
+If you are an experienced Vue developer, most of this application will be self-explanatory, though you can always refer back to this tutorial to understand the details.
+
+# Working with Vue
+
+The rest of this tutorial will take you on a step-by-step journey through the details of using Vue with Meteor.
+
+### Install Vue dependencies
+
+To start working with Vue as our view library, we first need to add the relevant npm package.
+
+Open a new terminal in the same directory as your running app, and type:
+
+```sh
+meteor npm install --save vue
+```
+
+> Note: `meteor npm` supports the same features as `npm`, though the difference can be important. Consult the [`meteor npm` documentation](https://docs.meteor.com/commandline.html#meteornpm) for more information.
```

Comment:
Might be a good idea to write this differently. The command `npm` is just available in case a person has a local installation of `nodejs`, but I'd rather reason this by using the version of npm which is bundled with the meteor installation, which makes it the same version for everyone on this project.

Reply:
thanks for the feedback :) let me take some time to update the markdown files this weekend and then push up my changes. But if anything else comes up in any of the files let me know. I am excited to make this as accurate as possible.